### DISCUSSION TITLE
Add to App Store to be able to add to iCloud's profile download history

### DISCUSSION THREAD

**Discussion by josdyr** [linearmouse/linearmouse#617] [General]
Add to App Store to be able to add to iCloud's profile download history. In case you get a new device, you can easily look up all your apps.

**Comment by lujjjh** [comment]
I believe the App Store [will reject LinearMouse](https://pilotmoon.com/blog/2020/12/09/scroll-reverser-1-8#fn:2:~:text=We%27ve%20completed%20the,be%20rejected.). Alternatively, we could enable users to save their configuration files in iCloud.
